Dockerfile

```diff
@@ -27,10 +27,10 @@ RUN apt-get update -y && apt-get install -y \
 ##############
 
 RUN mkdir /opt/hisat2/ \
-    && wget ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads/hisat2-2.0.5-Linux_x86_64.zip \
-    && unzip -d /opt/hisat2/ hisat2-2.0.5-Linux_x86_64.zip \
-    && ln -s /opt/hisat2/hisat2-2.0.5/hisat2 /usr/bin/hisat2 \
-    && rm hisat2-2.0.5-Linux_x86_64.zip
+    && wget ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads/hisat2-2.1.0-Linux_x86_64.zip \
+    && unzip -d /opt/hisat2/ hisat2-2.1.0-Linux_x86_64.zip \
+    && ln -s /opt/hisat2/hisat2-2.1.0/hisat2 /usr/bin/hisat2 \
+    && rm hisat2-2.1.0-Linux_x86_64.zip
```

Comment:
This is about the point where I start wanting to be more careful about using version tags for these Docker images, so we don't break/alter existing processing profiles. (That isn't something to worry about in this PR, but we could tag the new version after it's merged.)

Reply:
Agreed. This is where I start asking about naming conventions too. In the absence of better processing-profile-like tracking, should I be naming this workflow more descriptively? (i.e. workflow_rnaseq_firststrand_hisat2.1.0_kallisto0.43_stringtie1.3.3.cwl). Especially since my next push is going to be a CWL for unstranded data?

Reply:
Genome's CwlPipeline stores a copy of the CWL and a reference to the Docker image, so from that perspective as long as the Docker image is tagged with a version, then the CWL filename can be simpler.

We do have to be aware of which versions a particular version of CWL is compatible with (when breaking parameter changes are introduced, for example), but I'm not sure what a good way to capture that would be.  In the long run hopefully we get away from these giant Docker images and the versions are captured by the `DockerRequirement` of the steps.